### PR TITLE
whitelist rpy2

### PR DESCRIPTION
**Pull request by ericvergnaud**
## Changes
whitelist rpy2

### Linked issues
Progresses #1901 

### Functionality 
None

### Tests
- [x] manually tested

### CODE REVIEW

File: src/databricks/labs/ucx/source_code/known.json [hunk 1]
```diff
@@ -20614,6 +20614,130 @@
...
+    "rpy2.robjects.lib.tidyr": [],
+    "rpy2.robjects.methods": [],
+    "rpy2.robjects.numpy2ri": [],
+    "rpy2.robjects.packages": [],
+    "rpy2.robjects.packages_utils": [],
+    "rpy2.robjects.pandas2ri": [],
+    "rpy2.robjects.robject": [],
+    "rpy2.robjects.vectors": [],
+    "rpy2.situation": [],
+    "rpy2.tests": [],
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/known.json:20670]: `rpy2.tests` are irrelevant for the production code

**Review by nfx — CHANGES_REQUESTED** [review]
remove test deps

File: src/databricks/labs/ucx/source_code/known.json (hunk 1 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json:20670]: done

**Review by nfx — APPROVED** [review]
lgtm

**Pull request merged**